app/recipe/tests/test_ingredients_api.py

```diff
@@ -0,0 +1,51 @@
+from django.test import TestCase
```

Comment:
According to the exercise definition there should be no ingredients API separated from the recipes one. So we don't really need those tests.

Reply:
I know I was just practising for fun 😂